### PR TITLE
Kaizen dApp Staking

### PR DESCRIPTION
**Pull request by ayumitk**
**Pull Request Summary**

- Fetch campaign data and show latest 5 dApps on the portal
- Add a modal to insert dApps short description
- Update styling

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

Screenshot 2023-11-01 at 7 16 10 PM

Screenshot 2023-11-10 at 8 41 47 AM

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Looks great!!
- I tried to `position: fixed;` but again some oder changes and could not quite work it out, please first change this to review properly?
- Could we remove the arrow on the left when it is starting position and cannot go to the left please? and show this after it went to the right?
- Could we have transparent gradients that the card disappears by the maximum content width please?

<br />

Screenshot 2023-11-01 at 23 00 48


- I will work on some general contents for this new section
- This section has been a pain - could you change the section to fit in the width and tidy up the style? and thanks for making this area as same card style, this area also be updatable with Json file? I'll find new image for those area. 

<br />

Screenshot 2023-11-01 at 22 52 55

**Comment by ayumitk** [comment]
@Kahonnohak I've fixed what you pointed out above. Could you review the PR again?

**Comment by Kahonnohak** [comment]
Thank you!

Few things, sorry it was all in Figma but cannot see them unless preview it.

- background - lets discuss when your voice is back!!
- please remove the hover from those two areas (they are not buttons)

Screenshot 2023-11-03 at 13 00 14

Screenshot 2023-11-03 at 13 00 04

- I see shadow masked
Screenshot 2023-11-03 at 13 28 04
Screenshot 2023-11-03 at 14 32 36

- Could you apply boxshadow`0 0 16px 2px #00000014` for those two above please 
- Please also change the white to FFFFFF80 for the background, and remove boxshadow and hover from the content boxes.
Screenshot 2023-11-03 at 14 38 54

- Please also change this area
Screenshot 2023-11-03 at 14 40 22

**Comment by ayumitk** [comment]
@Kahonnohak I've fixed what you pointed out above and set `background-repeat: round;` for the background. Could you review the PR again?

**Review by bobo-k2 — APPROVED** [review]

File: src/components/assets/Account.vue [hunk 1]
```diff
@@ -3,13 +3,12 @@
     <div class="container">
       <div class="row--details">
         <div class="column-account-name">
-          <au-icon
+          <!-- <au-icon
```
Comment (impelcrypto) [inline src/components/assets/Account.vue:6]: @ayumitk @bobo-k2 
I think we don't need to comment out this. Just use the same code as [main](https://github.com/AstarNetwork/astar-apps/blob/main/src/components/assets/Account.vue#L6). What do you think?

Comment (bobo-k2) [reply src/components/assets/Account.vue:6]: I commented this out because CI was failing otherwise. Let me uncomment and see what will happen.

**Review by Kahonnohak — APPROVED** [review]
Great work!!

File: src/components/assets/Account.vue (hunk 1 above)
Comment (bobo-k2) [reply src/components/assets/Account.vue]: @impelcrypto fixed in https://github.com/AstarNetwork/astar-apps/pull/1001/commits/4c651d07be88409610e123d3f5a1c6b2dd403b9d

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**